trk/src/Screens/ClimbDetail/index.js

```diff
@@ -155,6 +155,11 @@ function ClimbDetail(props) {
     navigation.navigate('Feedback', { climbName: climbData.name, climbGrade: climbData.grade, climbId: climbId })
   };
 
+  const onDefinition = (descriptor) => {
+    navigation.navigate('Definition', { descriptor: descriptor });
+  };
+  
+
   const getSelectedIndex = (value) => {
```

Comment:
this is the function that handle clicking on a desciptor term -> it navigates to glossaryDefinition and props the descriptor term